CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 🚨📢 v0.140.0
 
+⚠️ Major release ⚠️ - Please test thoroughly before / after upgrading.
```

Comment:
At this amount of small but breaking changes, it's probably better to warn people to treat this as a major release.